meson.build

```diff
@@ -106,5 +106,6 @@ endif
 
 # --------------------- Bindings
 if get_option('with_pybind11')
+  subproject('pybind11')
```

Comment:
... but I'm not sure I see any point to this. You don't need to force the subproject to be used, it will already be automatically used unless pybind11 is found on the system with a suitable version matching the optional `version: '>='X.Y` kwargs.

If you just want to generally force the use of vendored dependencies / subprojects, then meson has a global option for this, `-Dwrap_mode=forcefallback`. It is preferrable to rely on that option, which you can set in the project() function's default_options, because it allows third-party builders to override it if they really prefer to use their own packaged versions of pybind11, e.g. linux distributions will greatly appreciate that flexibility. And yes, they can override the default_options by simply manually passing -Dwrap_mode=default (or nofallback) on the command-line, no patching of the build system needed.

So leaning on the global wrap_mode option allows everyone to be happy.

Reply:
Ah, makes sense, `wrap_mode` sounds like the way to go. Updated.